book/modules/appendix1-exercises.tex

```diff
@@ -1,7 +1,57 @@
-
 \begin{exercises}
 
 	\begin{problist}
-		\prob XXX Finish
+		\prob For each equation given below, determine if it is linear. If so, write it in its standard form. If not, explain what makes it nonlinear.
+    \begin{enumerate}
+      \item $\cos(4)x_1+\mathrm{e}y_2+\pi z_3=\mathrm{e}^{\pi}$
+      \item $4x_1+2x_2+5x_4=4x_2+4x_5+5$
+      \item $12x+3xy+5z=2$
+      \item $\sin(4)x+\cos(4)y+\tan(4)z=\operatorname{sec}(4)x+\operatorname{cosec}(4)y+\operatorname{cot}(4)z$
+      \item $5x+2y+8z=\cos(y)$
+    \end{enumerate}
+    
+    \prob For each system of linear equations given below, (a) write down its augmented matrix; (b) use row reduction algorithm to determine if it is consistent or not; (c) for each consistent system, give the complete solution set.
+    \begin{enumerate}
+      \item \systeme{-10x_1-4x_2+4x_3=28, 3x_1+x_2-x_3=-8, x_1+x_2-\frac{1}{2}x_3=-3}
+      \item \systeme{3x_1-2x_2+4x_3=54, 5x_1-3x_2+6x_3=88, x_1=-3}
+      \item \systeme{x_1+2x_2+4x_3-3x_4=0, 3x_1+5x_2+6x_3-4x_4=1, 4x_1+5x_2-2x_3+3x_4=3}
+      \item \systeme{x_1-x_2+5x_3+x_4=1, x_1+x_2-2x_3+3x_4=3, 3x_1-x_2+8x_3+x_4=5, x_1+3x_2-9x_3+7x_4=5}
+    \end{enumerate}
+    
+    \prob Let $\mathcal{A}$ be a system of linear equations with $n$ unknowns and $m$ equations. Kokoro is studying the condition for $\mathcal{A}$ to be consistent. Below, let $\operatorname{Coe}(\mathcal{A})$ be the coefficient matrix of \(\mathcal{A}\), and let $\operatorname{Aug}(\mathcal{A})$ be the augmented matrix of \(\mathcal{A}\).
```

Comment:
This setup is too much and introducing new notation is confusing. I also worry that it's too abstract. Can you craft something involving specific 2x2 and 2x3 matrices (you could but the occasional unknown in a coefficient matrix)?